### PR TITLE
Java: Move ping() commands out of baseclient

### PR DESCRIPTION
**Pull request by acarbonetto**
Move the `ping()` command out of baseclient and into the redisclient.  Javadocs updated appropriately and copied more from Python.

### CODE REVIEW

**Review by SanHalacogluImproving — APPROVED** [review]

File: java/client/src/main/java/glide/api/commands/ConnectionManagementCommands.java [hunk 1]
```diff
@@ -14,17 +14,16 @@ public interface ConnectionManagementCommands {
      * Ping the Redis server.
      *
      * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
-     * @return Response from Redis containing a <code>String</code> with "PONG".
+     * @return <code>String</code> with "PONG".
```
Comment (Yury-Fridlyand) [inline java/client/src/main/java/glide/api/commands/ConnectionManagementCommands.java:17]: Time to polish the doc!
```suggestion
     * @return <code>String</code> with <code>PONG</code>.
```
or
```suggestion
     * @return <code>String</code> with a server response.
```
(update all docs if you start)

**Review by Yury-Fridlyand — APPROVED** [review]
Add UT for ping with route for coverage

File: java/client/src/main/java/glide/api/commands/ConnectionManagementCommands.java (hunk 1 above)
Comment (acarbonetto) [reply java/client/src/main/java/glide/api/commands/ConnectionManagementCommands.java:17]: I can do the first... ;)

**Comment by acarbonetto** [comment]
> Add UT for ping with route for coverage

ping_with_route and ping_with_message_with_route already exist.

**Comment by Yury-Fridlyand** [comment]
> > Add UT for ping with route for coverage
> 
> ping_with_route and ping_with_message_with_route already exist.

It is IT, not UT

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request merged**